pkg/helpers/klusterletconfig.go

```diff
@@ -0,0 +1,32 @@
+package helpers
+
+import (
+	"fmt"
+
+	listerklusterletconfigv1alpha1 "github.com/stolostron/cluster-lifecycle-api/client/klusterletconfig/listers/klusterletconfig/v1alpha1"
+	klusterletconfighelper "github.com/stolostron/cluster-lifecycle-api/helpers/klusterletconfig"
+	klusterletconfigv1alpha1 "github.com/stolostron/cluster-lifecycle-api/klusterletconfig/v1alpha1"
+	"github.com/stolostron/managedcluster-import-controller/pkg/constants"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+)
+
+func GetMergedKlusterletConfigWithGlobal(
```

Comment:
Do we need to rename to GetKlusterletConfig.

Reply:
Kindly note the name should retain the 'Global' part for the sake of readability.